src/layouts/default/PlayerOSD/Player.vue

```diff
@@ -134,6 +123,8 @@ defineProps<Props>();
 
 // global refs
 const theme = useTheme();
+// Custom breakpoint for compatibility with `getBreakpointValue`. Can replace once we switch to using built-in Vuetify breakpoints
```

Comment:
Oh yeah, I looked into this a couple of weeks ago and its a complete mystery to me why we aren't just using default vuetify for this. My best guess is that this was not yet available back then in the vuetify beta's.

To me, we should now go (back) to using as much native vuetify as possible and just follow along

Reply:
Agreed. I did try going straight to using the vuetify config with this change, but the breakpoints are out-of-sync for some things, causing some buttons to be display and others not at certain widths. We should do one single change to all of them at the same time to move to the Vuetify config to better catch any bugs that might crop up. I've slightly started this work in https://github.com/JeffersonBledsoe/music-assistant-frontend/tree/get-breakpoint-tidy-1